sellotape/sellotape_dj/chat/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```

Comment:
This is what we call "הכנה למזגן"... if you don't plan to introduce tests on this PR (although you probably should), this file is not needed.